tests/unit/test_apigateway.py

```diff
@@ -366,3 +372,55 @@ def test_render_custom_template(self, template):
             "stageVariable1": "value1",
             "stageVariable2": "value2",
         }
+
+
+def test_import_swagger_api():
+    api_spec = load_test_resource("openapi.swagger.json")
+    api_spec_dict = json.loads(api_spec)
+
+    # patch integration model
+    apigateway_starter.apply_patches()
```

Comment:
Not sure we want to establish this as a pattern - most of our patches are currently written in a non-idempotent way and calling them directly from a unit test may have undesired side effects. We usually write unit tests only for generic logic that does not depend on any of the actual service providers.

Let's better move this test to `tests/integration/...` where we can safely assume that the patches have already been properly applied.. 👍